### PR TITLE
Created Filter page

### PR DESCRIPTION
**Pull request by Sharmaa-Ansh**

### CODE REVIEW

File: Components/Filter/Filter.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+width: 1rem;
+}
+
+.category-heading{
+    font-weight: 500;
+    font-size: 1.3rem;
+    margin: 1rem 1rem;
+}
+
+li{
```
Comment (ravikumar1002) [inline Components/Filter/Filter.css:31]: don't select direct HTML tag

**Review by ravikumar1002 — APPROVED** [review]
suggestion:  - try to using HTML semantic elements (like nav, header, aside, main)
                     -  and you should also give details in description box (like ss or short video) 

 Your naming convention and the class name you are using is really awesome.
looks good to merge

File: Components/Filter/Filter.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    <link rel="stylesheet" href="https://iconica.netlify.app/Components.css">
+    <link rel="stylesheet" href="/Components/Filter/Filter.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="navigation nav-color">
+        <div class="navigation_Title"> Iconica Store</div>
+        <div class="searchbar"><input type="text" placeholder="Search"></div>
+        <div class="navigation_Interaction">
+            <a class="navigation__login" href="/Components/Login and Signup/Login.html">Login</a>
```
Comment (Parul1999) [inline Components/Filter/Filter.html:16]: Space b/w Folder names is not recommended

File: Components/Filter/Filter.html [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+            <a href="/Components/Wishlist/Wishlist.html"><i  class="far fa-heart"></i></a>
+            <i class="fas fa-shopping-cart"></i>
+        </div>
+    </div>
+    <div class="page__container">
+        <div class="filter__sidebar">
+            <div class="category-heading">Filters</div>
```
Comment (Parul1999) [inline Components/Filter/Filter.html:23]: Try using semantic tags for the headings

File: Components/Filter/Filter.html [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+                            alt="">
```
Comment (Parul1999) [inline Components/Filter/Filter.html:76]: Write what is the image about in alt

Comment (Sharmaa-Ansh) [reply Components/Filter/Filter.html:76]: noted

File: Components/Filter/Filter.html (hunk 3 above)
Comment (Sharmaa-Ansh) [reply Components/Filter/Filter.html:23]: noted

File: Components/Filter/Filter.html (hunk 2 above)
Comment (Sharmaa-Ansh) [reply Components/Filter/Filter.html:16]: noted

File: Components/Filter/Filter.css (hunk 1 above)
Comment (Sharmaa-Ansh) [reply Components/Filter/Filter.css:31]: noted

**Pull request merged**